test/loki_service.cc

```diff
@@ -327,27 +330,17 @@ const std::vector<std::pair<uint16_t, std::string>> osrm_responses{
     {400,
      R"({"code":"InvalidValue","message":"The successfully parsed query parameters are invalid."})"}};
 
-zmq::context_t context;
```

Comment:
the diff looks odd here because i pulled the configuration making out into its own function. this part of the test and the tests that rely on it didnt change